hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/config/HandlersChainProperties.java

```diff
@@ -0,0 +1,57 @@
+package pl.allegro.tech.hermes.frontend.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import pl.allegro.tech.hermes.frontend.publishing.handlers.HandlersChainParameters;
+
+@ConfigurationProperties(prefix = "frontend")
+public class HandlersChainProperties {
+
+    private boolean keepAliveHeaderEnabled = false;
+
+    private int keepAliveHeaderTimeoutSeconds = 1;
+
+    private boolean authenticationEnabled = false;
+
+    private String authenticationMode = "constraint_driven";
```

Comment:
minor but imo `authenticationEnabled` and `authenticationMode` can be wrapped inside small class